app/people/customer.py

```diff
@@ -0,0 +1,8 @@
+# from app.cinema.hall import CinemaHall
+class Customer:
+    def __init__(self, name: str, food: str) -> None:
+        self.name = name
+        self.food = food
+
+    def watch_movie(self, movie: str) -> None:
+        print(f'{self.name} is watching \"{movie}\".')
```

Comment:
fix everywhere 
```suggestion
        print(f'{self.name} is watching "{movie}".')
```
or 
```suggestion
        print(f"{self.name} is watching \"{movie}\".")
```